### PR TITLE
[Task]api-doc add field and detail crud(#4029)

### PR DESCRIPTION
**Pull request by ywj1352**
Make sure that:

- [ ] You have read the [contribution guidelines](https://shenyu.apache.org/community/contributor-guide).
- [ ] You submit test cases (unit or integration tests) that back your changes.
- [ ] Your local test passed `./mvnw clean install -Dmaven.javadoc.skip=true`.

### CODE REVIEW

File: shenyu-admin/src/main/java/org/apache/shenyu/admin/model/query/FieldQuery.java [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+    /**
+     * the field desc.
+     */
+    private String fieldDesc;
+
+
+    /**
+     * the ext.
+     */
+    private String ext;
```
Comment (dragon-zhang) [inline shenyu-admin/src/main/java/org/apache/shenyu/admin/model/query/FieldQuery.java:40]: please remove `ext` condition, it is appropriate to use ES, but ES is too complex, so this condition is abandoned directly.

Comment (dragon-zhang) [inline shenyu-admin/src/main/java/org/apache/shenyu/admin/model/query/FieldQuery.java:40]: please remove `ext` condition, it is appropriate to use ES, but ES is too complex, so this field is abandoned directly.

File: shenyu-admin/src/main/java/org/apache/shenyu/admin/controller/FieldController.java [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+     *
+     * @param name        name
+     * @param fieldDesc   fieldDesc
+     * @param ext         ext
+     * @param currentPage currentPage
+     * @param pageSize    pageSize
+     * @return {@linkplain ShenyuAdminResult}
+     */
+    @GetMapping("/findPageByQuery")
+    public ShenyuAdminResult listByPage(final String name, final String fieldDesc, final String ext,
```
Comment (dragon-zhang) [inline shenyu-admin/src/main/java/org/apache/shenyu/admin/controller/FieldController.java:85]: please remove `ext` condition.

File: shenyu-admin/src/main/java/org/apache/shenyu/admin/model/query/FieldQuery.java (hunk 1 above)
Comment (ywj1352) [reply shenyu-admin/src/main/java/org/apache/shenyu/admin/model/query/FieldQuery.java:40]: done

Comment (ywj1352) [reply shenyu-admin/src/main/java/org/apache/shenyu/admin/model/query/FieldQuery.java:40]: done

File: shenyu-admin/src/main/java/org/apache/shenyu/admin/controller/FieldController.java (hunk 2 above)
Comment (ywj1352) [reply shenyu-admin/src/main/java/org/apache/shenyu/admin/controller/FieldController.java:85]: done

**Review by dragon-zhang — APPROVED** [review]

**Pull request merged**